tests/sentry/tasks/post_process/tests.py

```diff
@@ -236,36 +235,12 @@ def test_simple(self):
                 tags=[('foo', 'bar'), ('biz', 'baz')],
             )
 
-        tags = list(EventTag.objects.filter(
-            event_id=event.id,
-        ).values_list('key_id', 'value_id'))
-        assert len(tags) == 2
-
-        tagkey = tagstore.get_tag_key(
-            project_id=self.project.id,
-            environment_id=self.environment.id,
-            key='foo',
-        )
-        tagvalue = tagstore.get_tag_value(
-            project_id=self.project.id,
-            environment_id=self.environment.id,
-            key='foo',
-            value='bar',
-        )
-        assert (tagkey.id, tagvalue.id) in tags
-
-        tagkey = tagstore.get_tag_key(
-            project_id=self.project.id,
-            environment_id=self.environment.id,
-            key='biz',
-        )
-        tagvalue = tagstore.get_tag_value(
-            project_id=self.project.id,
-            environment_id=self.environment.id,
-            key='biz',
-            value='baz',
-        )
-        assert (tagkey.id, tagvalue.id) in tags
```

Comment:
This was reaching into tagstore too much so I tried to make it validate that `index_event_tags` was doing the right thing without literally checking rows in the database (which is tagstore's responsibility, in my opinion.)

There are tests confirming this behavior for v2, but not for legacy (there are actually *no* tests for legacy, except those that are covered by integration tests, sort of like this one.)